tweet_processing.py

```diff
@@ -0,0 +1,71 @@
+import numpy as np 
+import pandas as pd 
+import spacy
+import sklearn.feature_extraction.text as text
+import matplotlib.pyplot as plt
+import re
+
+data = []
+cols = ["tweets"]
+
+df1 = pd.read_csv("sarw.csv")
+df2 = pd.read_csv("info.csv")
+df3 = pd.read_csv("uscg.csv")
+df4 = pd.read_csv("last_week.csv")
+
```

Comment:
I can't run this without these CSVs

Please (a) add them in if they're going to be used in the project, or (b) just the concatenated one on line 16